docs-v2/lib/dirToTree.js

```diff
@@ -35,30 +38,33 @@ const createFile = (stat, fileName, workingDir, baseDir) => {
       name,
       title,
       isFile: true,
-      route: path.join(routeBase, routeFile),
+      route: path.posix.join(routeBase, routeFile),
     }));
 };
 
 const createDirectory = (items, localWorkingDir, baseDir) => ({
-  route: localWorkingDir.replace(baseDir, ''),
+  route: localWorkingDir.replace(baseDir, '').replace(/\\/, '/'),
```

Comment:
comment that this is for windows compatibility, would be nice.

Reply:
sure